services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java

```diff
@@ -430,6 +438,13 @@ public BatchResults deleteLinkProps(List<LinkPropsDto> keys) {
         return Arrays.asList(updatedLink);
     }
 
+    @Override
+    public List<LinkDto> setLinkBfd(LinkEnableBfdDto link) {
```

Comment:
Would be great if you add logging for this request
`log.debug("Setting BDF ....");`